minard/burst.py

```diff
@@ -0,0 +1,52 @@
+import couchdb
+from . import app
+
+def load_burst_runs(offset):
+    """
+    Returns a dictionary with the burst runs loaded from couchdb. The dummy itterator is used as key
+    to keep the ordering from the couchdb query. The content of the couchdb document is stored as values.
+    This loads ALL the documents in burst database, ordered by run_subrun_burst logic.
+    The logic to limit and split the results per page was implemented.
+    """
+    server = couchdb.Server("http://snoplus:"+app.config["COUCHDB_PASSWORD"]+"@"+app.config["COUCHDB_HOSTNAME"])
+    db = server["burst"]
+
+    i = 0 #the counter keeps the ordering correct
+    results = {}
+    limit = 10    #this changes the # of lines per page, can be modified when the DB gets big
```

Comment:
Would be nice to add this as a query to the page, i.e.

```python
@app.route('/burst')
def burst():
    offset = request.args.get('offset',default=0,type=int)
    limit = request.args.get('limit',default=10,type=int)

    return render_template('burst.html',data=burst_f.load_burst_runs(offset)[0], total=burst_f.load_burst_runs(offset)[1], offset=burst_f.load_burst_runs(offset)[2], limit=burst_f.load_burst_runs(offset)[3] )
```

and then pass `limit` to this function. The nice thing about this is that then you can just change the url to `snopl.us/monitoring/burst?limit=100` to change it.

Reply:
great, done!